### DISCUSSION TITLE
How to use mini-css-extract-plugin with webpack built libraries in a monorepo?

### DISCUSSION THREAD

**Discussion by charlesritchea** [webpack/webpack#18487] [Q&A]
I have a non-cli Angular yarn workspaces / lerna monorepo. There is a mixture of non-angular typescript libraries built with rollup and some "app module" **libraries with Angular components that are built with webpack**. Then of course the final app package built with webpack. I can get mini-css-extract-plugin to work with the final app package, but if I use it for the "app module" libraries then styles don't load nor are there 404s or anything indicating missing files. Is this even supported? Does the final app package put dependency styles in it's extracted css (I dont' think so)

**Comment by alexander-akait** [comment]
I don't recommend to bundle libraries with CSS, just provide function/docs how to include styles in an application

**Comment by charlesritchea** [comment]
Angular components include their styles. We're talking >5000 files here, I
wouldn't know how to implement your suggestion if I could

**Comment by charlesritchea** [comment]
Also, there are no other consumers of these libraries. Instructions
wouldn't do anything. They are bundled in the final app bundle. It's a
large private monorepo that doesn't publish libraries just uses them for
modularity
